src/WebUI/Pods/PodsTable.tsx

```diff
@@ -26,21 +27,36 @@ export interface IPodsTableProperties extends IVssComponentProperties {
     podsToRender: V1Pod[];
     headingText?: string;
     nameFilter?: string;
+    showWorkloads?: boolean;
 }
 
 export class PodsTable extends BaseComponent<IPodsTableProperties> {
+    private statusCount: { [key: string]: number } = {};
     public render(): React.ReactNode {
         const filteredPods: V1Pod[] = this.props.podsToRender.filter((pod) => {
             return Utils.filterByName(pod.metadata.name, this.props.nameFilter);
         });
 
         if (filteredPods.length > 0) {
+            filteredPods.forEach(pod => {
+                const key = (pod.status.message ? pod.status.reason : pod.status.phase).toLowerCase();
+                if (key in this.statusCount) {
```

Comment:
if (key in this.statusCount) {
                    this.statusCount[key] += 1;
                } else {
                    this.statusCount[key] = 1;
                }

i did not get the use of this

Reply:
@there is a subtext below "Associated Pods" title on the lines of `X unschedulable, Y running" 
So we are concatenating all the statuses together.